### PR TITLE
View & BE category

### PR DESCRIPTION
**Pull request by khieutrung12**
## Related Tickets:

-   [#46241](https://edu-redmine.sun-asterisk.vn/projects/ecommerce-intern-hn/issues/46241)

## WHAT this PR does?

- View manage category
- Crud category

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ready

**Review by DTHNhung — APPROVED** [review]

File: resources/lang/en/titles.php [hunk 1]
```diff
@@ -71,4 +75,9 @@
     'File extension: ' => 'File extension: ',
     'Choose a photo' => 'Choose a photo',
     'Hello' => 'Hello',
+=======
```
Comment (ghost) [inline resources/lang/en/titles.php]: chưa fix hết conflict nè, xem lại nha

File: resources/lang/vi/titles.php [hunk 2]
```diff
@@ -25,14 +25,17 @@
...
+    'slug' => 'Slug',
     'add-brand' => 'Thêm thương hiệu',
     'all-brand' => 'Thương hiệu',
     'brand-name' => 'Tên thương hiệu',
     'add' => 'Thêm',
     'update' => 'Cập nhật',
+    'delete' => 'Xóa',
     'add-var' => 'Thêm :name',
     'update-var' => 'Cập nhật :name',
+<<<<<<< HEAD
```
Comment (ghost) [inline resources/lang/vi/titles.php]: đây nữa

**Comment by khieutrung12** [comment]
ready

**Review by DTHNhung — APPROVED** [review]

File: resources/views/admin/category/edit_category.blade.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                                        {{ __($message, ['name' => __('titles.name-var', ['name' => __('titles.category')])]) }}</span>
+                                @enderror
+                            </div>
+                            <div class="form-group">
+                                <label for="category_id">
+                                    {{ __('titles.parent-category') }}</label>
+                                <select class="form-control select2"
+                                    name="parent_id" id="parent_id">
+                                    <option value="">
+                                        None
```
Comment (ghost) [inline resources/views/admin/category/edit_category.blade.php]: localization

File: resources/views/admin/category/index_row.blade.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        {{ $category->slug ?? '' }}
+    </td>
+    <td>
+        <a href="{{ route('categories.edit', $category->id) }}"
+            class="active styling-edit" ui-toggle-class="">
+            <i class="fas fa-edit text-success text-active"></i>
+        </a>
+        <form action="{{ route('categories.destroy', $category->id) }}"
+            method="POST"
+            onsubmit="return confirm('{{ __('messages.confirmDelete', ['name' => __('titles.category')]) }}');"
```
Comment (ghost) [inline resources/views/admin/category/index_row.blade.php]: bỏ code onsubmit ra file js riêng

File: resources/views/admin/category/index_row.blade.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            style="display: inline-block;">
```
Comment (ghost) [inline resources/views/admin/category/index_row.blade.php]: bỏ style ra css riêng

File: resources/views/admin/category/index_row.blade.php [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+            @csrf
+            @method('DELETE')
+            <button type="submit" class="delete-icon"
+                onclick="return confirm('{{ __('messages.confirmDelete', ['name' => __('titles.category')]) }}')">
```
Comment (ghost) [inline resources/views/admin/category/index_row.blade.php]: tương tự với event onclick, tạo js riêng cho file

**Comment by khieutrung12** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**